google/__init__.py

```diff
@@ -28,4 +28,10 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 # pylint: disable=missing-docstring
-__import__('pkg_resources').declare_namespace(__name__)  # pragma: no cover
+from __future__ import absolute_import
```

Comment:
The actual problem, incidentally, was `tox.ini` setting `{toxinidir}` in the `PYTHONPATH`.